garnett/trajectory.py

```diff
@@ -1276,30 +1300,24 @@ def _generate_type_id_array(types, type_ids):
     return _type
 
 
-def copyto_hoomd_blue_snapshot(frame, snapshot=None):
+def _to_hoomd_snapshot(frame, snapshot):
     "Copy the frame into a HOOMD-blue snapshot."
-    if frame.position is not None:
-        np.copyto(snapshot.particles.position, frame.position)
-    if frame.orientation is not None:
-        np.copyto(snapshot.particles.orientation, frame.orientation)
-    if frame.velocity is not None:
-        np.copyto(snapshot.particles.velocity, frame.velocity)
-    if frame.mass is not None:
-        np.copyto(snapshot.particles.mass, frame.mass)
-    if frame.charge is not None:
-        np.copyto(snapshot.particles.charge, frame.charge)
-    if frame.diameter is not None:
-        np.copyto(snapshot.particles.diameter, frame.diameter)
-    if frame.moment_inertia is not None:
-        np.copyto(snapshot.particles.moment_inertia, frame.moment_inertia)
-    if frame.angmom is not None:
-        np.copyto(snapshot.particles.angmom, frame.angmom)
-    if frame.image is not None:
-        np.copyto(snapshot.particles.image, frame.image)
+    for prop in FRAME_TRAJ_PROPS[4:]:
+        if getattr(frame, prop) is not None:
+            np.copyto(getattr(snapshot.particles, prop), getattr(frame, prop))
     return snapshot
 
 
-def copyfrom_hoomd_blue_snapshot(frame, snapshot):
+@deprecation.deprecated(deprecated_in="0.7.0",
+                        removed_in="0.8.0",
+                        current_version=__version__,
+                        details="This function is deprecated.")
+def copyto_hoomd_blue_snapshot(frame, snapshot):
+    "Copy the frame into a HOOMD-blue snapshot."
+    return _to_hoomd_snapshot(frame, snapshot)
+
+
+def _from_hood_snapshot(frame, snapshot):
```

Comment:
```suggestion
def _from_hoomd_snapshot(frame, snapshot):
```
(This probably needs tests...)